### PR TITLE
001 employee response controller service

### PR DESCRIPTION
**Pull request by johnhutchman**

### CODE REVIEW

File: views/salesEmployee.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <body>
+        <h1>Employee</h1>
+        <table>
+            <tr>
+                <th>Employee ID</th>
+                <th>Employee Name</th>
+            </tr>
+            {% for employee in salesEmployee | sort(attribute='id') %}
+                <tr>
+                    <td><a href="/employees/sales/{{employee.employeeId}}" target="_blank">{{ employee.id }}</a></td>
```
Comment (mlynch147) [inline views/salesEmployee.html]: There's no view to display this... remove the hyperlink

File: views/delliveryEmployee.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <body>
+        <h1>Employee</h1>
+        <table>
+            <tr>
+                <th>Delivery ID</th>
+                <th>Delivery Name</th>
+            </tr>
+            {% for employee in deliveryEmployee | sort(attribute='id') %}
+                <tr>
+                    <td><a href="/employees/delivery/{{employee.employeeId}}" target="_blank">{{ employee.id }}</a></td>
```
Comment (mlynch147) [inline views/delliveryEmployee.html]: There's no view to display this... remove the hyperlink

File: views/delliveryEmployee.html
Comment (mlynch147) [inline views/delliveryEmployee.html:1]: rename to deliveryEmployeeList.html

File: views/salesEmployee.html
Comment (mlynch147) [inline views/salesEmployee.html:1]: rename to salesEmployeeList.html

**Review by mlynch147 — CHANGES_REQUESTED** [review]

**Review by mlynch147 — APPROVED** [review]

**Review by TaylorB-softdevkainos — APPROVED** [review]

**Review by mlynch147 — APPROVED** [review]

**Pull request merged**